### PR TITLE
ZCS-8012 : Upgrade jquery in zm-admin-console

### PR DESCRIPTION
**Pull request by auminpatel**
- Moving jquery and clipboard code to its own modules
- Not minifying the 3rd party code using yiucomposser instead using the minified versionthat is shipped with the 3rd party lib
- Loading Jquery is loaded before itself has it is happening now
- Upgrading jquery version to 3.3.1-1

### CODE REVIEW

**Comment by auminpatel** [comment]
Deleting this branch as removing the jquery from admin-console

File: build.xml [hunk 1]
```diff
@@ -1035,31 +1036,44 @@ ext = BeanUtils.cook(ext);
...
+                                <echo>Copying thirdparty library file to @{var}${jsmin.ext}</echo>
+                                <copy tofile="@{var}${jsmin.ext}" file="@{var}"/>
+                            </then>
+                            <else>
+                                <try>
+                                    <block>
+                                        <echo>Minimizing to @{var}${jsmin.ext}</echo>
+                                        <java jar="${build.tmp.dir}/yuicompressor-2.4.8.jar" fork="true" failonerror="true"
+                                            output="@{var}${compressoutput.ext}">
+                                            <!-- Add error="@{var}${compresserror.ext}" in java task if *.err files need to be generated. -->
```
Comment (BhavinBathani) [inline build.xml]: Do we need this commented code?

File: build.xml [hunk 2]
```diff
@@ -1035,31 +1036,44 @@ ext = BeanUtils.cook(ext);
...
-		                    </finally>
-	                    </try>
+                        <if>
+                            <or>
+                                <equals arg1="@{var}" arg2="${build.js.dir}/JQuery_all.js" />
+                                <equals arg1="@{var}" arg2="${build.js.dir}/Clipboard_all.js" />
+                            </or>
+                            
+                            <then>
+                                <echo>Copying thirdparty library file to @{var}${jsmin.ext}</echo>
```
Comment (BhavinBathani) [inline build.xml]: Do we need to add jquery word in this echo/log message?

Comment (auminpatel) [reply build.xml]: Have added this for proper message for during building admin-console

File: build.xml (hunk 1 above)
Comment (auminpatel) [reply build.xml]: For future reference the message was kept , so that if we want to generate error files ,lets keep it.

**Pull request merged**